agents/hermes/plugin/__init__.py

```
@@ -3,22 +3,962 @@
+        proc = subprocess.Popen(
+            [sys.executable, "-c", _BROWSER_USE_CDP_TUNNEL_SCRIPT, cdp_url],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.DEVNULL,
+            text=True,
+            env=os.environ.copy(),
+        )
+        port = proc.stdout.readline().strip() if proc.stdout else ""
+        if not port.isdigit():
+            proc.terminate()
+            return cdp_url
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)